ci-operator/templates/cluster-launch-e2e-azure.yaml

```diff
@@ -0,0 +1,200 @@
+  required: true
+- name: CLUSTER_TYPE
+  value: "azure"
+  required: true
+- name: TEST_SUITE
+- name: TEST_FOCUS
+- name: TEST_SKIP
+  value: "\\[local\\]"
+- name: TEST_SUITE_SERIAL
+- name: TEST_FOCUS_SERIAL
+- name: TEST_SKIP_SERIAL
+  value: "\\[local\\]"
+- name: TEST_COMMAND
+
+objects:
+
+# We want the cluster to be able to access these images
+- kind: RoleBinding
+  apiVersion: authorization.openshift.io/v1
+  metadata:
+    name: ${JOB_NAME_SAFE}-image-puller
+    namespace: ${NAMESPACE}
+  roleRef:
+    name: system:image-puller
+  subjects:
+  - kind: SystemGroup
+    name: system:unauthenticated
+
+# The e2e pod spins up a cluster, runs e2e tests, and then cleans up the cluster.
+- kind: Pod
+  apiVersion: v1
+  metadata:
+    name: ${JOB_NAME_SAFE}
+    namespace: ${NAMESPACE}
+    annotations:
+      # we want to gather the teardown logs no matter what
+      ci-operator.openshift.io/wait-for-container-artifacts: teardown
+  spec:
+    restartPolicy: Never
+    activeDeadlineSeconds: 10800
+    terminationGracePeriodSeconds: 600
+    volumes:
+    - name: artifacts
+      emptyDir: {}
+    - name: shared-tmp
+      emptyDir: {}
+    - name: cluster-profile
+      secret:
+        secretName: ${JOB_NAME_SAFE}-cluster-profile
+
+    containers:
+
+    # Once admin.kubeconfig exists, executes shared tests
+    - name: test
+      image: ${IMAGE_TEST}
+      resources:
+        requests:
+          cpu: 1
+          memory: 300Mi
+        limits:
+          cpu: 3
+          memory: 4Gi
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      - name: cluster-profile
+        mountPath: /tmp/cluster
+      - name: artifacts
+        mountPath: /tmp/artifacts
+      env:
+      - name: HOME
+        value: /tmp/home
+      - name: TEST_COMMAND
+        value: "${TEST_COMMAND}"
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        set -euo pipefail
+
+        trap 'touch /tmp/shared/exit' EXIT
+        trap 'kill $(jobs -p); exit 0' TERM
+
+        cp "$(which oc)" /tmp/shared/
+
+        mkdir -p "${HOME}"
+
+        # wait until the setup job creates admin.kubeconfig
+        while true; do
+          if [[ ! -f /tmp/shared/admin.kubeconfig ]]; then
+            sleep 15 & wait
+            continue
+          fi
+          break
+        done
+        echo "Found shared kubeconfig"
+
+        # don't let clients impact the global kubeconfig
+        cp /tmp/shared/admin.kubeconfig /tmp/admin.kubeconfig
+        export KUBECONFIG=/tmp/admin.kubeconfig
+
+        PATH=/usr/libexec/origin:$PATH
+
+        
+        # create the environment, but exit before we run tests
+        #TODO: RUN TESTS HERE
+        exit 0
+
+    # Runs an install
+    - name: setup
+      image: ${IMAGE_TEST}
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp
+      - name: cluster-profile
+        mountPath: /go/src/github.com/openshift/openshift-azure/injected
```

Comment:
Where this comes from?

Reply:
Mount to a different location; openshift/openshift-azure is confusing. I would probably use `/var/azure/credentials` or `/etc/azure/credentials`